signals/signals.go

```diff
@@ -0,0 +1,20 @@
+package signals
```

Comment:
why aren't just using `signal.NotifyContext`?

Reply:
I thought that came out in 1.18 🤦🏻 thanks.

Reply:
Resolved.